Objects/EventItem/Chicken.gd

```diff
@@ -2,3 +2,11 @@ extends "res://Objects/EventItem/EventItem.gd"
 
 func _ready():
 	set_start_position(600, 293)
+
+func _process(delta):
+	position.x += move_speed * delta
```

Comment:
Moving should be processed automatically because almost each event object moves. Will talk about it in the incoming meeting.